ai_review/analyzers/metrics.py

```
@@ -0,0 +1,64 @@
+
+        # keep filenames that _don't_ match the ignore regex
+        if files and isinstance(files, list):
+            for r in compiled_patterns:
+                if platform == 'github':
+                    files = [f for f in files if (f.filename and not r.match(f.filename))]
+                elif platform == 'bitbucket':
+                    # files = [f for f in files if (f.new.path and not r.match(f.new.path))]
+                    files_o = []
+                    for f in files:
+                        if hasattr(f, 'new'):
+                            if f.new and f.new.path and not r.match(f.new.path):
```

Ruff (0.8.2)

Use a single `if` statement instead of nested `if` statements

(SIM102)